docs/learning-the-basics.md

```diff
@@ -51,7 +51,7 @@ You can customize the dock's behavior through **[System Settings](/docs/learning
 
 Apps exist in their own windows which can be closed, maximized, or moved around.
 
-When you open an app, its window appears on the desktop. Each app typically has three areas: the window buttons, a toolbar, and the contents of the app.
+When you open an app, its window appears on the desktop. Each app typically has three areas: The window buttons, a toolbar, and the contents of the app.
```

Comment:
Do not capitalize lists after colons. See: https://www.grammarly.com/blog/capitalization-after-colons/